### DISCUSSION TITLE
Default Settings for Categories in Preferences

### DISCUSSION THREAD

**Discussion by jmarbutt** [magicbell/community#153] [Questions] [answered]
Currently when creating a category, all users are subscribed to to that event if they receive it. 

Is it possible to allow some categories to be optional?

For example, say I create a category of `unassigned_task` by default I may want all users to receive that in the app but not via a text message. If a user wants to receive that by text, great but don't make it the default.

**Comment by unamashana** [comment, marked as answer]
Hey @jmarbutt we don't offer a way to change this behavior, but it does sound useful, and we will add to our product backlog for consideration. Thanks!

**Reply by jmarbutt** [reply]
Thanks that would be great because there are a lot of "optional" type notifications that more power users would like but the average user may not want turned on specifically.
